modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb

```diff
@@ -229,7 +229,185 @@ class Form526ResponseSwagger
                   items do
                     key :type, :string
                     key :example, 'Hardship'
-                    key :enum, ['Hardship', 'Seriously Injured/Very Seriously Injured', 'Terminally Ill', 'Homeless', 'Purple Heart', 'POW', 'Medal of Honor', 'Amyotrophic Lateral Sclerosis', 'Emergency Care']
+                    key :enum, %w[
+                      1151 P&T
+                      38 USC 1151
+                      Active Chapter 31 Case
+                      Agent Orange - Vietnam
+                      Amyotrophic Lateral Sclerosis
+                      Annual IU Eligibility Review-Fraud
+                      Attorney Fee
+                      Authzn to Release Information to Third Party
+                      Automobile Adaptive Equipment Paid
+                      Automobile Allowance Paid
+                      BVA Decision
+                      Blind
+                      Blue Water Navy
+                      C-123
+                      CADD - Written Requests Only
+                      CAVC Temp File
+                      CH31 SAM
+                      CHAMPUS Entitlement Exists
+                      CRSC/CRDP Case
+                      CSRS
+                      CWA Exposure OIF/OND
+                      Case Managed Development
+                      Clear and Unmistakable Error
+                      Closed Chapter 31 Case
+                      Committee on Waiver Case
+                      Compensation Award Line
+                      Compensation Under 1925 Rating Schedule
+                      Congressional
+                      Converted - Automobile Adaptive Equipment Paid
+                      Converted - Automobile Allowance Paid
+                      Converted - Blind
+                      Converted - Comp Under 1925 Rating Schedule
```

Comment:
I'm pretty sure using %w will actually end up converting this into "Converted", "-", "Comp", "Under"... Can you double check and confirm for me in a test that this won't happen?